Rename 'Checking account' page to 'Savings account.' (#314)
BitcoinDesign/Guide: guide/case-studies/savings-account.md

-->

## Savings account

> In this section, we are looking at a product that is meant to be a replacement for what a bank would call a savings account where the user might receive their salary, pay their bills from, or store wealth long term. Safeguards against loss will be a higher priority than with a cash product, and we might therefore accept more friction both when setting up the wallet and when transacting. If users have no prior bitcoin knowledge we should expect to spend a significant effort educating them to put them in a position to safely operate the wallet product.

Reviewer: ```suggestion
In this section, we are looking at a product that is meant to be a replacement for what a bank would call a savings account where the user might store wealth long term. Safeguards against loss will be a higher priority than with a frequent spending product, and we might therefore accept more friction both when setting up the wallet and when transacting. If users have no prior bitcoin knowledge we should expect to spend a significant effort educating them to put them in a position to safely operate the wallet product.
```

Reviewer: This case study was not written to describe a Savings account (you wouldn't expect to pay your bills from this account). But re-reading it, and as it suggests a multi-key setup I agree it might be better as a Savings account.

Made some changes to remove references to salary and bills.

Reviewer 2: Found the term "Transaction account" on [Wikipedia](https://en.wikipedia.org/wiki/Transaction_account). How about using that term as it would also allow us to link to the Wikipedia page as a further reference?

"[Deposit](https://en.wikipedia.org/wiki/Deposit_account)" account is also a thing, although Wikipedia warns that that's different across countries. Also, interesting overview of [account types](https://en.wikipedia.org/wiki/Bank_account#Types_of_accounts).

Reviewer: Yes, if the content were to stay the same 'Transaction account' would be more appropriate.

I am now thinking that a multi-key setup might be overkill for a transactional account though. 
A single key wallet with discovery protection (I.e hardware wallet(s) with pass-phrase) would be sufficient.

Author: Transaction account sounds more like the daily spending case study. Yeah I thought the multi-key parts were definitely more in line with savings
